.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,5 +12,5 @@ Fixes / closes #????
 
 <!-- If this is a feature pull request / breaks compatibility: -->
 <!-- (Otherwise, remove these lines.) -->
-- [ ] Corresponding documentation was added / updated.
+- [ ] Corresponding documentation was added / updated (including any warning- and error-messages added / removed / modified).
```

Comment:
You're the native English speaker, not me, but I'm surprised by the use of dashes here.

Reply:
I think it works fine either way; I'll update it.